sphinx/theming.py

```diff
@@ -211,7 +216,7 @@ def _load_theme_with_ancestors(
         if inherit not in theme_paths:
             msg = __(
                 'The %r theme inherits from %r, which is not a loaded theme. '
-                'Loaded themes are: %s',
+                'Loaded themes are: %s'
             ) % (name, inherit, ', '.join(sorted(theme_paths)))
```

Comment:
As you can see, '%' is put on the same line as ')' so I don't know why it cannot do it when possible.